dadi/lib/cors/index.js

```diff
@@ -0,0 +1,56 @@
+const path = require('path')
+
+const config = require(path.join(__dirname, '/../../../config.js'))
+const vary = require('vary')
+
+module.exports = function cors (server) {
+  server.app.use((req, res, next) => {
+    if (config.get('cors') !== true) {
+      return next()
+    }
+
+    const method = req.method && req.method.toUpperCase && req.method.toUpperCase()
```

Comment:
Is it necessary to check for the existence of both `req.method` and `req.method.toUpperCase` before using it? Would `req.method` ever be anything other than a string?

Reply:
also, we lowercase it everywhere else ;)

Reply:
😃Fair enough! Just pushed a less conservative check.